portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -651,6 +651,13 @@ definition {
 			radioOption = "${textFieldLineOption}");
 	}
 
+	macro viewAnyField {
```

Comment:
Good job, @Jamersonze! Let's just rename this macro and the associated path removing the word **any**, e.g. let's rename to `viewField` and `FormFields#FIELD`.

Reply:
Renamed